mlx_engine/model_kit.py

```diff
@@ -100,23 +108,38 @@ def __init__(
             )
 
     @staticmethod
-    def _validate_kv_cache_quantization_params(
+    def _set_kv_cache_quantization_params(
```

Comment:
```suggestion
    def _get_kv_cache_quantization_params(
```
As far as I can tell this doesn't actually set anything but rather does a light round of processing on `kv_bits, kv_group_size, quantized_kv_start` to get the final params for usage

Reply:
Renamed to `_get_kv_cache_quantization_params`